app/lib/sidebar/local_widgets/list_tile.dart

```diff
@@ -1,47 +1,55 @@
 import 'package:app/navbar/navigation.dart';
+import 'package:app/style/style.dart';
 import 'package:flutter/material.dart';
 
-class CustomTile extends StatelessWidget {
+class TemplateListTile extends StatelessWidget {
```

Comment:
Template is literally my favourite word because of the way it's used in c++ applies to so many things